dashboard/src/components/CreateActionModal.js

```diff
@@ -33,7 +33,10 @@ const CreateActionModal = ({ person = null, persons = null, isMulti = false, com
 
   const onAddAction = async (body) => {
     if (body.status !== TODO) body.completedAt = body.completedAt || Date.now();
-    const response = await API.post({ path: '/action', body: prepareActionForEncryption(body) });
+    const response = await API.post({
+      path: '/action',
+      body: prepareActionForEncryption({ ...body, user: body.user || user._id }),
+    });
```

Comment:
pas besoin de vérifier à la création: le `user` est toujours précisé

https://github.com/SocialGouv/mano/blob/0c5e9a75a72bed25d926eb11b21687862cef8f01/dashboard/src/components/CreateActionModal.js#L79-L91

```suggestion
const response = await API.post({ path: '/action', body: prepareActionForEncryption(body) });
```